.drone.star

```diff
@@ -163,52 +118,29 @@ def check_starlark(ctx, trigger = {}, depends_on = []):
                 },
             },
         ],
-        "depends_on": depends_on,
         "trigger": trigger,
-    }
+    }]
 
-def build_and_test_client(ctx, c_compiler, cxx_compiler, build_type, generator, trigger = {}, depends_on = []):
+def ctest_pipeline(ctx, c_compiler, cxx_compiler, build_type, generator, trigger = {}):
```

Comment:
I think unit_test_pipeline will be nice name to this method, as it does unit tests